R/derive_extreme_records.R

```diff
@@ -95,6 +95,12 @@
 #'
 #'   *Permitted Values*: a condition
 #'
+#' @param keep_source_vars Variables to be kept in the new records
+#'
+#'   A named list or tidyselect expressions returned by `exprs()` defining the
```

Comment:
```suggestion
#'   A named list or tidyselect expressions created by `exprs()` defining the
```

To match most of the rest of documentation phrasing